### PR TITLE
docs: add example for: awscc_controltower_enabled_control

### PR DESCRIPTION
**Pull request by gokendra1**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

**Comment by wellsiau-aws** [comment]
@Gokendra , thank you for your contribution.

instead of adding the md file directly, could you use the `.md.tmpl` template file instead

you can find example of similar pull request here: https://github.com/hashicorp/terraform-provider-awscc/pull/1343/files

File: examples/resources/awscc_controltower_enabled_control/controltower_enabled_control.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+
+
+# The following resource enables the control 'AWS-GR_AUDIT_BUCKET_RETENTION_POLICY'
+
+resource "awscc_controltower_enabled_control" "example" {
+  control_identifier = "arn:aws:controltower:us-east-1::control/AWS-GR_AUDIT_BUCKET_RETENTION_POLICY"
+  target_identifier  = "arn:aws:organizations::000123456789:ou/o-ydkk7uvfj1/ou-e3wm-2x760yqd"
```
Comment (wellsiau-aws) [inline examples/resources/awscc_controltower_enabled_control/controltower_enabled_control.tf:8]:
```suggestion
  target_identifier  = "arn:aws:organizations::<<your-account-id>:ou/<<your-org-id>>/<<your-ou-id>>"
```

File: examples/resources/awscc_controltower_enabled_control/controltower_enabled_control.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+}
+
+# Please change the Organization ID from 'o-ydkk7uvfj1' to your Organization ID, and the Organizational Unit ID from 'u-e3wm-2x760yqd' to your desired OU where the controls need to be implemented.
```
Comment (wellsiau-aws) [inline examples/resources/awscc_controltower_enabled_control/controltower_enabled_control.tf:11]:
```suggestion
# Please change the Organization ID to your Organization ID, and the Organizational Unit ID to your desired OU where the controls need to be implemented.
```

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
minor tweak requested, please have a look

**Review by wellsiau-aws — APPROVED** [review]
LGTM

**Comment by marcosentino** [comment]
@gokendra1 Thank you for the contribution!

**Pull request merged**